steps/cli/oc.go

```diff
@@ -165,6 +166,34 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify versioned ecosystem tasks", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	requiredTasks := []string{"buildah", "git-cli", "git-clone", "kn", "kn-apply", "maven", "openshift-client", "s2i-dotnet", "s2i-go", "s2i-java", "s2i-nodejs", "s2i-perl", "s2i-php", "s2i-python", "s2i-ruby", "skopeo-copy", "tkn"}
+	requiredVersions := []string{"1-16-0", "1-17-0"}
```

Comment:
You cannot hard-code versions here, that would increase maintenance burden. Just read the expected version from env variable `OSP_VERSION` as defined `env/default/default.properties`. You need to transform the string but that is not a big deal. It's enough to verify current version, you don't need to check all versions.

Reply:
Done